packages/experience/src/App.tsx

```diff
@@ -61,7 +63,7 @@ const App = () => {
                     element={<SocialSignInWebCallback />}
                   />
                   <Route path="direct/:method/:target?" element={<DirectSignIn />} />
-
+                  <Route path="token/:token" element={<OneTimeToken />} />
```

Comment:
You can either reuse the direct/:method path or consider using a more specific one, such as one-time-token. Additionally, since the token itself is considered a secret value instead of ID, it might be more appropriate to pass it as a query parameter rather than including it in the path.

Reply:
Updated to `/one-time-token/:token`